### PR TITLE
Better error handling

### PR DESCRIPTION
**Pull request by matdenkas**

### CODE REVIEW

File: src/ModelExecution/modelWrapper.py [hunk 1]
```diff
@@ -101,9 +101,18 @@ def make_and_save_prediction(self, dateTime: datetime) -> any:
         """
         try:
             inputs = self.__inputGatherer.get_inputs(dateTime)
-            if len(inputs) == 0: return -1
+            if len(inputs) == 0: 
+                log('No inputs received for model.')
```
Comment (abeasley1722) [inline src/ModelExecution/modelWrapper.py:105]: why return -1 instead of returning log?

Comment (bestrada33) [reply src/ModelExecution/modelWrapper.py:105]: It always returned -1, he just added proper logging to the error now. Before it didn't tell you anything

**Review by bestrada33 — APPROVED** [review]
LGTM

**Review by lovelysandlonelys — APPROVED** [review]
looks good to me

**Review by abeasley1722 — APPROVED** [review]

**Pull request merged**